zingoconfig/src/lib.rs

```diff
@@ -1,4 +1,8 @@
+//! ZingConfig
```

Comment:
@Oscar-Pepper can you please add a description of what this crate is, here?    No need for perfection, just enough to give a reader a general idea.  Fine to include `TODO`s for enhancements to the doc-comment.